src/messages.cpp

```diff
@@ -224,13 +224,15 @@ const char* FC msgSec( 	// get sub-msg text for system error code
 		{ SECBADFN, "invalid file name" },
 		{ SECBADRV, "invalid drive letter" },
 		{ SECOTHER, "unknown error, errno not set" },
-		{ 32767,    "" }			// terminator
+		{ 32767,    "unhandle system error" }			// terminator
 	};
 
-	const char *pText				// pt to text corresponding to SEC
-	  =  sec >= 0 && sec < sys_nerr 	// for errors in system range
+	const char *pText =				// pt to text corresponding to SEC
+		// sys_nerr and sys_errlist are deprecated
+		// https://github.com/jgbit/vuda/issues/11
+	    /* sec >= 0 && sec < sys_nerr 	// for errors in system range
```

Comment:
1.  Better to recode rather than comment out.  If old code is to be left, better to use #if 0 rather than comment.  Comments are for comments.
2.  Based on a quick read, the web page cited here indicates that there are newer facilities that perhaps replace the deprecated stuff.  What capabilities are being lost by just eliminating use of the deprecated stuff?  Is it worth the work to reimplement those capabilities using the new stuff?
3.

Reply:
The "newer facilities" come with their own safety concerns. We weren't able to find a clean replacement that didn't have issues of its own.